### PR TITLE
fix: update warning message if optional parameters only

### PR DESCRIPTION
**Pull request by yuqizhou77**
example: 
(!) Warning: There is no required parameter defined in API "searchProducts", so we set the first optional parameter as the parameter for command "searchProducts". Mitigation: If "image_url" is not what you need, please edit the parameter of command "searchProducts" in "appPackage\manifest.json". The parameter name should match what's defined in "appPackage\apiSpecificationFiles\openapi.json".

### CODE REVIEW

**Review by nliu-ms — DISMISSED** [review]

**Review by SLdragon — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -257,8 +257,8 @@
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.mitigation": "Mitigation: update \"%s\" field in \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate": "Missing \"%s\" in command \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate.mitigation": " Mitigation: create Adaptive Card template in \"%s\" and then update \"%s\" field to the relative path in \"%s\".",
-  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCommandParameters": "Missing parameters in command \"%s\".",
-  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCommandParameters.mitigation": " Mitigation: add at least one parameter for command \"%s\" in \"%s\". The parameter name should match what's defined in \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly": "There is no required parameter defined in API \"%s\", so we set the first optional parameter as the parameter for command \"%s\".",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:260]: recommend:
There is no required parameter defined in API \"%s\". The first optional parameter is set as the parameter for command \"%s\".

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -257,8 +257,8 @@
...
+  "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly.mitigation": " Mitigation: If \"%s\" is not what you need, please edit the parameter of command \"%s\" in \"%s\". The parameter name should match what's defined in \"%s\".",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:261]: can remove 'please'

Comment (sffamily) [reply packages/fx-core/resource/package.nls.json:261]: we standardized to not use 'please' in messages. and no "I" or "We" as well.

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (yuqizhou77) [reply packages/fx-core/resource/package.nls.json]: Thanks, updated.

**Review by sffamily — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Pull request merged**